### ISSUE TITLE
怪异

### ISSUE THREAD

**Issue by THUdsg** [Faustinaqq/AF-CLIP#2]
state_x = torch.cat([self.state_prompt_embedding, state_x], dim=1)[:, :77, :] 

是这样拼的，很怪。按理说应该把可学习提示插在sos和eos之间。

**Comment by invincible-xiaobing** [comment]
我也注意到这点了，这么做有什么说法吗

**Comment by Linda6521** [comment]
请问复现的环境是什么，好像复现出来和原文差距很大
